lib/accesslint/ci/scanner.rb

```diff
@@ -0,0 +1,42 @@
+require "uri"
+
+module Accesslint
+  module Ci
+    class Scanner
+      def self.perform(*args)
+        new(*args).perform
+      end
+
+      def initialize(host:, options: {})
+        @host = host
+        @options = options
+      end
+
+      def perform
+        `#{command}`
```

Comment:
Had to look up backticks vs. `Kernel.system` vs. `Open3`, this looks like the best approach.